docs/getting_started/go/first_program_in_go/index.md

```diff
@@ -607,9 +607,8 @@ You can view more information about the process in the [Temporal Web UI](localho
 
 ![The next Activity](images/activity_failure.png)
 
-Click the **Stack Trace** link to see a stack trace showing you the errors, as well as details about the pending Activity:
-
-![The stack trace of the Activity](images/stack_trace.png)
+Click any ActivityTaskFailed event link to see Failure detail information and a Call Stack trace.
```

Comment:
```suggestion
Click any `ActivityTaskFailed` event link to see the Failure details and call stack.
```

NOTE: My suggested wording better matches what is in the current UI, although the screenshot used in the tutorial is outdated.

Reply:
Thank you. Adopted.

Reply:
Thank you, adopted.